components/Typography/src/MDCTypography.m

```diff
@@ -185,45 +186,135 @@ + (UIFont *)boldFontFromFont:(UIFont *)font {
 
 @end
 
+@interface MDCSystemFontLoader ()
+
+/**
+ Basic cache to keep track of recently-used fonts.
+ */
```

Comment:
Am not certain that this comment is describing anything more than what the variable name does.